prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.html

```diff
@@ -1,4 +1,4 @@
-<app-enrollee-page [busy]="busy">
+<app-enrollee-page [busy]="{busy, message: NG_BUSY_SUBMISSION_MESSAGE}">
```

Comment:
So the reason I asked you not to do this ticket for all submissions until the first ticket was reviewed was to allow feedback and avoid making changes across the application that might be unnecessary, as well as, tickets being scored, prioritized top down, and above the ready line regardless of Kanban or Sprint when pulled from the Backlog.

There may be no need to pass the message in through the configuration.  You could likely create a service within NgxBusyModule with show/hide methods that you can pass in the message from your observable using a side-effect.  You might not even need a hide method depending on the components lifecycle.  You'll need to address the opacity in a way that doesn't affect the whole app adversely, but improves readability for this use case.

Reply:
At the time when I picked up this ticket, it was already on top of the other tickets above the **Ready** line and was scored. Please check the ticket history on Jira.

The reason I brought the work for all submissions together in this PR was because it was the same work for all _overview_ components. I will revert to add this only for user enrolment and bring the same work over to the other ticket. In any way, I don't see the ng-busy causing a break in the app since it doesn't interfere with any part of it other than displaying the spinner asynchronously.

Reply:
After refinement it wasn't put at the bottom and that's good feedback.  

No it doesn't break the app but you didn't need to make these changes if you used a service which is why I wanted the review on it.